### ISSUE TITLE
pageBreak after every row???

### ISSUE THREAD

**Issue by aldin94** [simonbengtsson/jsPDF-AutoTable#387]
i have a table and want to print every row on a new page. i tried it with drawRow() but i didnt find a solution. is it possible? 

Here is my code: 

            var doc = new jsPDF('p', 'pt');
            var startingPage = doc.internal.getCurrentPageInfo().pageNumber;

            var d = new Date();
            var month = d.getMonth()+1;
            var day = d.getDate();
            var output = d.getFullYear() + '/' + (month<10 ? '0' : '') + month + '/' + (day<10 ? '0' : '') + day;

            var res = doc.autoTableHtmlToJson(document.getElementById("export"));
            doc.autoTable(res.columns, res.data, {margin: {top: 80}});

            var header = function(data) {
                doc.setFontSize(18);
                doc.setTextColor(40);
                doc.setFontStyle('normal');
                //doc.addImage(headerImgData, 'JPEG', data.settings.margin.left, 20, 50, 50);
                doc.text("Export", data.settings.margin.left, 50);
            };

            var options = {
                addPageContent: header,
                margin:
                {
                    top: 80
                },
                headerStyles: {
                    fillColor: [192, 192, 192],
                    textColor: [0, 0, 0],
                    halign: 'left'
                },
                showHeader: 'everyPage',
                pageBreak: 'always',
            };

            doc.autoTable(res.columns, res.data, options);

            doc.save("druckansicht_" + output);

**Comment by simonbengtsson** [comment]
It should be possible with the `willdrawCell` hook in v3 together with doc.addPage().

**Comment by kurthybela** [comment]
Hi. How can i reset the row's height on the new page in the willDrawCell hook? Currently the row is at the end of the page, because the row y position does not change. I want to insert a pagebreak and insert the last row on the next page, if the summary lines (footer) do not fit on the current page. I would like to insert the column labels also on the new page. So there should be at least one table row if the summary (below the table) needs to be on the new page. Is it possible? Thanks

**Comment by donqq** [comment]
> It should be possible with the `willdrawCell` hook in v3 together with doc.addPage().

I have tried this method, and found that it caused the autotable to print the row in the next page without resetting its position, for instance let's say there are 10 rows? and all fits in the a page? 1st row is printed on 1th place, 2nd row is printed in 2nd place etc.. likewise 6th row should be printed in the 6th place. If I used WilldrawCell hook and doc.addPage() function to print the 7th row in the next page, it still prints in the 7th place in the next page instead of in the 1st place in the next page. The position is not reset. I also experienced some weird black colour in the first cell since 2nd page. How could I fix this problem?
